config/development.json

```diff
@@ -123,6 +123,7 @@
 		"marketplace-fetch-all-dynamic-products": true,
 		"marketplace-personal-premium": false,
 		"marketplace-test": true,
+		"marketplace-reviews-show": true,
```

Comment:
I've created a PR adding this flag: https://github.com/Automattic/wp-calypso/pull/84524

The only differences are its encapsulated on its own PR and its added on all relevant files (stage, production). 

Maybe we should use the other PR as base for this one.